appyters/GSEA_Appyter/appyter.json

```diff
@@ -1,14 +1,22 @@
 {
   "$schema": "https://raw.githubusercontent.com/MaayanLab/appyter-catalog/master/schema/appyter-validator.json",
   "name": "GSEA_Appyter",
```

Comment:
The URL will be GSEA_Appyter.. we should change the directory & name if we wish it to be `AGSEA_Appyter`..

Reply:
Ah, ok. I was getting a version update earlier when I tried to change it to AGSEA_Appyter, though, so should the new version start again at 0.0.1?

Reply:
@cindyxio it should work as long as all files are in the new directory and no files are left in the old directory.